### PR TITLE
Clean up list. (Revised)

### PR DESCRIPTION
**Pull request by Unkn0wnCat**
I've noticed [some checks failing in travis](https://travis-ci.org/awesome-selfhosted/awesome-selfhosted/builds/644472339), so I am doing some cleanup in the commits included in this PR.

My previous PR was not quite passing yet, so this one is the revised one.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ before_script:
 script:
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then git diff origin/master -U0 README.md | grep -Pos "(?<=^\+).*" >> temp.md; fi || (exit 0)'
   - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then node tests/test.js temp.md; else node tests/test.js README.md; fi'
-  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then if [ -f temp.md ]; then awesome_bot temp.md --allow-redirect --skip-save-results --allow 202; else (exit 0); fi else awesome_bot README.md --allow-redirect --skip-save-results --allow 202; fi'
+  - 'if [ "$TRAVIS_PULL_REQUEST" != "false" ]; then if [ -f temp.md ]; then awesome_bot temp.md --allow-redirect --skip-save-results --allow 202 --white-list airsonic.github.io/docs/apps,github.com/bblfsh/bblfshd; else (exit 0); fi else awesome_bot README.md --allow-redirect --skip-save-results --allow 202 --white-list airsonic.github.io/docs/apps,github.com/bblfsh/bblfshd; fi'
```
Comment (nodiscc) [inline .travis.yml]: Please don't whitelist bblfshd, the duplicate Source Code link should be removed

File: README.md [hunk 2]
```diff
@@ -1478,7 +1478,7 @@ See also [Documentation Generators](#documentation-generators), [Wikimatrix](htt
 - [jingo](https://github.com/claudioc/jingo) - Git based wiki engine written for node.js, with a decent design, a search capability and good typography. `MIT` `Nodejs`
 - [Mediawiki](https://www.mediawiki.org/wiki/MediaWiki) - MediaWiki is a free and open-source wiki software package written in PHP. It serves as the platform for Wikipedia and the other Wikimedia projects, used by hundreds of millions of people each month. ([Demo](https://en.wikipedia.org/wiki/Main_Page), ([Source Code](https://phabricator.wikimedia.org/diffusion/MW/)) `GPL-2.0` `PHP`
 - [MoinMoin](https://moinmo.in/) - Advanced, easy to use and extensible WikiEngine with a large community of users. ([Source Code](http://hg.moinmo.in/moin)) `GPL-2.0` `Python`
-- [Olelo](https://github.com/minad/olelo) - Olelo is a wiki that stores pages in a Git repository, supports many markup styles and has an extensible, hackable architecture. ([Demo](http://gitwiki.org/)) `MIT` `Ruby`
```
Comment (nodiscc) [inline README.md]: Olelo is actually unmaintained since 2017, it should be removed.

File: .travis.yml (hunk 1 above)
Comment (Unkn0wnCat) [reply .travis.yml]: I've adjusted README.md and .travis.yml to fix this.

File: README.md (hunk 2 above)
Comment (Unkn0wnCat) [reply README.md]: I've gone ahead and removed it.

**Comment by Unkn0wnCat** [comment]
Thanks, have a nice day! :)

**Pull request merged**